eclipse-che/che-plugin-registry
# feat(plugins): generate che-theia-plugin.yaml fragment in addition to meta.yaml for VS Code extensions

### What does this PR do?
meta.yaml has no field to describe preferences, it's not compliant with devfile 2.0 definition of containers, etc.
so, when generating yaml files from che-theia-plugins.yaml , here it generates `che-theia-plugin.yaml` file in addition to meta.yaml

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/19545


### How to test this PR?
Launch the build or look at the PR result and browse index.json, you will see plugin entries for VS Code extensions and it will contain links to `che-theia-plugin.yaml` fragment that you can also look

example: https://pr-check-933-alpine-che-plugin-registry.surge.sh/v3/plugins/redhat/java11/latest/che-theia-plugin.yaml


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.


Change-Id: I5718fde84620934c15764aa1edcbbea9e42eda70
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
